MAINTENANCE.md

```diff
@@ -208,3 +209,75 @@ Here are some of the shortcuts included in [`bashrc_for_gmt`](/admin/bashrc_for_
 - `vpngdbuild` and `vpdfdbuild` can be used to open the results from all failing image-based tests.
 - `view_png_failures_r` and `view_pdf_failures_r` can be used for view failures of the release build with a lag between
   opening each file.
+
+## Continuous Integration
+
+We use GitHub Actions Continuous Integration (CI) services to build and test the project on Linux, macOS and Windows.
+
+There are 11 configuration files located in `.github/workflows/`:
+
+1. `backport.yml` (Backports PRs into the 6.1 branch)
+
+  This workflow backports PRs labels by "backport 6.1" into the 6.1 branch.
```

Comment:
Need to indent by one more whitespace, otherwise, it won't be correctly rendered.
```suggestion
   This workflow backports PRs labels by "backport 6.1" into the 6.1 branch.
```

Reply:
The workflow description is a little misleading. The workflow will backport any PRs labeled with "backport xxx" to the "xxx" branch. So it could be applied to any branch.

The backport also has a big limitation: https://github.com/GenericMappingTools/gmt/issues/3827. Perhaps should also be documented here.